### DISCUSSION TITLE
changing root folder of the request

### DISCUSSION THREAD

**Discussion by hoavt-54** [aio-libs/aiohttp#9444] [Q&A]
I have an web application serving at the root folder like this

```
from aiohttp import web
app = web.Application()
sk.attach_to_app(app, "/socket")
app.add_routes([web.get('/', lambda _: web.FileResponse('./static/index.html'))])
app.router.add_static('/', path='./static', name='static')
logging.debug("Added all the static web files")
web.run_app(app, host='0.0.0.0', port=8765)
```
However request from the loadbalancer always having a prefix **/app-demo** which causes the server to response with 404. Is there any config in the web so that it understands that the base of the urls is **/app-demo** and serving with **/** instead?

Edit:
This is how it is done in Flask:
```
app = Flask(__name__)
app.register_blueprint(bp, url_prefix='/abc/123')

```

**Comment by Dreamsorcerer** [comment]
Off the top of my head, maybe just put all your logic into a sub-app which can be installed under that prefix. Slightly redundant, but I don't remember a way to specify a prefix as the root.

**Reply by hoavt-54** [reply]
Thank you. That works.
